.pre-commit-config.yaml

```diff
@@ -52,12 +52,18 @@ repos:
 
 - repo: local
   hooks:
-  - id: dr-style
-    name: dr-style
+  - id: dr-style-all
+    name: dr-style-all
     entry: python scripts/dr-style.py --Werror include test examples
     language: system
     pass_filenames: false
     always_run: true
+  - id: dr-style-include
+    name: dr-style-include
+    entry: python scripts/dr-style.py --Werror --include include/dr/mhp
```

Comment:
@BenBrock: This is how pre-commit runs the scan. If you change the path above to /include/dr, it would scan shp as well. Then you could rely on the pre-commit testing or you could execute the above command manually.

Reply:
Thanks, I will give this a try today.